R/geom-violin.R

```diff
@@ -183,15 +184,15 @@ GeomViolin <- ggproto("GeomViolin", Geom,
       quantile_grob <- if (nrow(both) == 0) {
         zeroGrob()
       } else {
-        GeomPath$draw_panel(both, ...)
+        inject(GeomPath$draw_panel(both, !!!params))
       }
 
       ggname("geom_violin", grobTree(
-        GeomPolygon$draw_panel(newdata, ...),
+        inject(GeomPolygon$draw_panel(newdata, !!!params)),
```

Comment:
8. Here is a downside of the method: we cannot simply pass dots here as GeomViolin recieves more paremters than GeomPolygon needs.